modules/migration-mtc-cr-manifests.adoc

```diff
@@ -0,0 +1,363 @@
+// Module included in the following assemblies:
+//
+// * migration/migrating_3_4/migrating-applications-with-cam-3-4.adoc
+// * migration/migrating_4_1_4/migrating-applications-with-cam-4-1-4.adoc
+// * migration/migrating_4_2_4/migrating-applications-with-cam-4-2-4.adoc
+
+[id="migration-mtc-cr-manifests_{context}"]
+= {mtc-short} custom resource manifests
```

Comment:
Are all of these user created objects? Or are some of them created by the migration controller itself?

Reply:
Some are user-created and some are created by kubernetes objects.